File: Examples/PHP/Files/GetListFilesFromThirdPartyStorage.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class Files {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function getListFiles() {
        $result = $this->storageApi->GetListFiles($Path = "Files", $storage = "MyDropboxStorage");
        print_r($result);
	}
}

$files = new Files();
$files->getListFiles();

?>
//ExEnd:

File: Examples/Ruby/Files/get_list_files_from_third_party_storage.rb
#ExStart:
require 'aspose_storage_cloud'

class Files
    
  include AsposeStorageCloud

  def initialize
    # Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("App Key", "App SID")
    @storage_api = StorageApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../../../data/" << file_name,"r") { |io| io.read } )
  end
  
  def get_list_files
    response = @storage_api.get_list_files("Files", {storage: "MyDropboxStorage"})
  end

end

files = Files.new()
puts files.get_list_files
#ExEnd: